tox.ini

```diff
@@ -85,15 +74,15 @@ commands = check-manifest
 
 [testenv:release]
 description = build the changelog, bump the package version, commit and tag
-basepython=python2.7
+basepython=python3.8
 skip_install = true
-whitelist_externals =
+allowlist_externals =
     git
     tox
 passenv =
     HOME
 deps =
-    bumpversion==0.5.3
+    bumpversion==0.6.0
```

Comment:
We should not bump the version ourselves. Let the author decide which version he wants to upgrade to after merging these changes.

Reply:
that's just the dependency not the package itself